cardea-data/src/main/java/ca/on/oicr/gsi/cardea/data/ShesmuSample.java

```diff
@@ -16,11 +13,13 @@ public class ShesmuSample {
 
   private final String id;
   private final boolean supplemental;
+  private final boolean qcFailed;
 
 
   private ShesmuSample(Builder builder) {
     this.id = requireNonNull(builder.id);
     this.supplemental = requireNonNull(builder.supplemental);
+    this.qcFailed = requireNonNull(builder.qcFailed);
```

Comment:
These `builder` fields are both `boolean`, which means they will default to `false` and can't be null. They should remain `boolean` in `ShesmuSample`, but should probably both be `Boolean` in the builder to ensure that the values are set properly and not defaulted

Reply:
I made the change and can fix the test np but thinking about it more I don't think it will ever default the way the code is setup now, and I don't think there should ever be a null?
What would a null mean?